applications/callflow/src/cf_util.erl

```diff
@@ -796,18 +796,11 @@ process_event(Call, NoopId, JObj) ->
 -spec get_timezone(kz_json:object(), kapps_call:call()) -> ne_binary().
 get_timezone(JObj, Call) ->
     case kz_json:get_ne_binary_value(<<"timezone">>, JObj) of
-        'undefined'   -> account_timezone(Call);
-        <<"inherit">> -> account_timezone(Call);  %% UI-1808
+        'undefined'   -> kz_account:timezone(kapps_call:account_id(Call));
+        <<"inherit">> -> kz_account:timezone(kapps_call:account_id(Call));
         TZ -> TZ
     end.
 
--spec account_timezone(kapps_call:call()) -> ne_binary().
-account_timezone(Call) ->
-    case kz_account:fetch(kapps_call:account_id(Call)) of
-        {'ok', AccountJObj} -> kz_account:timezone(AccountJObj);
-        {'error', _E} -> kz_account:default_timezone()
```

Comment:
Semantics are not preserved: this will now crash on error: https://github.com/2600hz/kazoo/blob/master/core/kazoo_documents/src/kz_account.erl?utf8=%E2%9C%93#L248

Was this change on purpose?